app/Http/Requests/Admin/WorkedHoursExportColumn/CreateWorkedHoursExportColumnRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Requests\Admin\WorkedHoursExportColumn;
+
+use App\Models\Field;
+use App\Models\WorkedHoursExportColumn;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateWorkedHoursExportColumnRequest extends FormRequest {
+	/**
+	 * Determine if the user is authorized to make this request.
+	 *
+	 * @return bool
+	 */
+	public function authorize() {
+		return $this->user()->can('create', WorkedHoursExportColumn::class);
+	}
+
+	/**
+	 * Get the validation rules that apply to the request.
+	 *
+	 * @return array
+	 */
+	public function rules() {
+		return [
+			'column' => 'required|string|unique:worked_hours_export_columns',
```

Comment:
both of these don't have to be unique I think. if they want repetitions they can have them

Reply:
also there should be validation that the column is legal